tests/test_data/test_datasets/test_vfi_dataset.py

```diff
@@ -54,8 +54,26 @@ def test_vfi_vimeo90k_dataset(self):
         data_infos = dataset.data_infos[0]
         assert_dict_has_keys(data_infos, ['inputs_path', 'target_path', 'key'])
 
+    def test_vfi_vimeo90k_7frames_dataset(self):
+
+        dataset_cfg = dict(
+            type='VFIVimeo90K7FramesDataset',
+            folder=self.folder,
+            ann_file=self.ann_file,
+            pipeline=self.pipeline,
+            input_frames=[1, 3, 5, 7],
+            target_frames=[4])
+        dataset = build_dataset(dataset_cfg)
+        data_infos = dataset.data_infos[0]
```

Comment:
we'd better test `dataset[0]` because this is the interface called by dataloader.